lexicons/tools/ozone/set/add.json

```diff
@@ -0,0 +1,31 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.sets.add",
+  "defs": {
+    "main": {
+      "type": "procedure",
+      "description": "Add values to a specific set",
+      "input": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["name", "values"],
+          "properties": {
+            "name": {
+              "type": "string",
+              "description": "Name of the set to add values to"
+            },
```

Comment:
Should we reflect the set name restrictions (length between `3` and `128`) here?

Reply:
this is only for specifying which set the values will be added to so the caller is expected to either know the name of the set or fetch the name in a previous call to `querySets` so the restriction on name doesn't make a lot of sense to me. wdyt?